ci-operator/config/openshift/installer/openshift-installer-fcos.yaml

```diff
@@ -27,19 +27,15 @@ base_images:
     name: "4.2"
```

Comment:
We should probably drop `origin_4.2_cli` too?  And `openshift_release_golang-1.13`?  Maybe more?  Not clear to me  if these are automatically audited for unused cruft or not.

Reply:
Not sure what's the process of dropping those. In any case I'd prefer to follow master here